app/components/status_tag/view.rb

```diff
@@ -13,7 +13,8 @@ def initialize(trainee:, classes: "")
   def status
     {
       submitted_for_trn: "pending trn",
-      recommended_for_qts: "qts recommended",
+      recommended_for_qts: "#{trainee.get_award_type.downcase} recommended",
```

Comment:
Maybe leave these as they are for now - I'm not sure we're certain yet whether we're going to add separate trainee states for `recommended_for_eyts` and `eyts_awarded`

Reply:
To be decided here: https://trello.com/c/CAtRXyah/1570-l-enable-award-of-eyts